docusaurus/docs/Android/04-compose/03-channel-components/04-default-channel-item.mdx

```diff
@@ -0,0 +1,203 @@
+# DefaultChannelItem
+
+The `DefaultChannelItem` component represents the `ChannelList` items that are shown by default, if you don't customize its `itemContent`.
+
+However, you can also use the `DefaultChannelItem`, and customize its Slot APIs, in case you want only replace a specific part of the default UI. 
+
+:::note 
+The `DefaultChannelItem` is great if you want default UI or replace specific slots, however, you can still completely override the `itemContent` parameter of the `ChannelList`, if you need even more custom UI.
+:::
+
+Let's see how to use and customize this component.
+
+## Usage
+
+To use the `DefaultChannelItem`, it's best to override the `itemContent` parameter in the `ChannelList` and use it for the channel items:
+
+```kotlin
+override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+
+    setContent {
+        val user by ... // fetch user from ChatDomain or ViewModel
+        
+        ChatTheme {
+            ChannelList(
+                itemContent = { channel -> // Customize the channel items
+                	DefaultChannelItem(
+                    	channel = channel,
+                    	currentUser = user,
+                        onChannelLongClick = { },
+                        onChannelClick = { },
+                	)
+            	}
+            )
+        }
+    }
+}
+```
+
+This is a very basic and crude example of a `ChannelList`, where you override the `itemContent`.
+
+The snippet above will generate the following UI.
+
+![The Default ChannelList Component](../../assets/compose_default_channel_list_component.png)
+
+In order to fully utilize the `DefaultChannelItem` let's see how to handle its actions and how to customize its Slot APIs.
+
+## Handling Actions
+
+The `DefaultChannelItem` signature exposes the following actions that can be intercepted:
+
+```kotlin
+@Composable
+fun DefaultChannelItem(
+	onChannelClick: (Channel) -> Unit,
+	onChannelLongClick: (Channel) -> Unit,
+    ... // state and other UI-related customization options 
+)
+```
+
+* `onChannelClick`: Handler for the user tapping on an item. Useful for starting the `MessagesScreen` in your app.
+* `onChannelLongClick`: Handler for the user long tapping on an item. Useful when you want to control how state changes or what UI you want to show, when the user selects an item in the list.
+
+Here's an example of using the default component, but overriding the behavior:
+
+```kotlin
+override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+
+    setContent {
+        ChatTheme {
+			ChannelList(
+				itemContent = { channel ->
+					DefaultChannelItem(
```

Comment:
Indentation is growing up. 😄

Reply:
Ugh I tried doing them with tabs specifically for this reason, but idk what's going on